frontend/src/lib/k8s/KubeObject.ts

```diff
@@ -74,7 +74,10 @@ export class KubeObject<T extends KubeObjectInterface | KubeEvent = any> {
   }
 
   constructor(json: T, cluster?: string) {
-    this.jsonData = json;
+    this.jsonData = {
+      ...json,
+      apiVersion: json?.apiVersion ?? (this.constructor as typeof KubeObject).apiVersion,
```

Comment:
jsondata is supposed to be the exact thing we receive from the backend and I think we should try to avoid modifying it. 
could you explain where in the code this field is needed and expected?

Reply:
The problem with the row actions is that AuthVisible requires a KubeObject and the jsondata provides an object without an apiVersion, which is needed for auth. Without the apiVersion, the row actions do not appear, so this is supposed to fix that. Will add this to the PR description